tla/consensus/MCccfraftAtomicReconfig.cfg

```diff
@@ -5,7 +5,7 @@ CONSTANTS
     Servers <- ToServers
 
     MaxTermLimit = 5
-    RequestLimit = 3
+    RequestLimit = 1
```

Comment:
Unfortunately necessary to continue to run within the time constraints of the CI, but @lemmy [ran the model checking](https://github.com/microsoft/CCF/pull/5973#issuecomment-1986368930) with the original parameters and checked they passed still.